lib/fluent/plugin/output.rb

```diff
@@ -534,6 +552,35 @@ def handle_stream(tag, es)
         meta_and_data.keys
       end
 
+      def handle_stream_with_standard_format(tag, es)
+        meta_and_data = {}
+        records = 0
```

Comment:
No.
If input event stream is an instance of `MessagePackEventStream` and it was instantiated without # of events, `es.events` will return 0. It may occur in case when input event stream is from `in_forward`, and incoming data(opt) doesn't have additional information about # of events.